src/Mod/Sketcher/Gui/ViewProviderSketch.cpp

```diff
@@ -2204,37 +2195,22 @@ void ViewProviderSketch::onSelectionChanged(const Gui::SelectionChanges& msg)
                         int GeoId = std::atoi(&shapetype[4]) - 1;
                         resetPreselectPoint();
                         preselection.PreselectCurve = GeoId;
-
-                        if (sketchHandler)
-                            sketchHandler->applyCursor();
-                        this->updateColor();
```

Comment:
This is already done in mouseMove

Reply:
It's done here: https://github.com/FreeCAD/FreeCAD/blob/ea6c4ea60fd998429108b864ea9a56e21b60df76/src/Mod/Sketcher/Gui/ViewProviderSketch.cpp#L741
which is called here:
https://github.com/FreeCAD/FreeCAD/blob/f9ba94c70b139ece6ba5ffba47632d91b5c9c645/src/Mod/Sketcher/Gui/DrawSketchController.h#L567

So while it's not done _in_ `mouseMove`, it is called after.